.github/workflows/gmud_pr_analysis.yml

```diff
@@ -0,0 +1,85 @@
+name: GMUD PR Analysis
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches: [main]
+
+jobs:
+  analyze:
+    if: github.event.pull_request.base.ref == 'main' && (github.event.pull_request.head.ref == 'dev')
+    name: Analyze GMUD PR
+    runs-on: ubuntu-latest
+    container: python:3.9-alpine
+    steps:
+      - name: Checkout repositories
+        uses: actions/checkout@v4
+        with:
+          repository: aziontech/azion-quality-index
+          ref: main
+          path: azion-quality-index
+          token: ${{ secrets.GLOBAL_TOKEN }}
+      
+      - name: Checkout target repository
+        uses: actions/checkout@v4
+        with:
+          path: target-repo
+        
+      - name: Set up environment
+        run: |
+          apk add --no-cache git
+          cd azion-quality-index
+          pip install -r requirements.txt || echo "No requirements.txt found"
+          pip install PyGithub jira
```

Comment:
Why not keep this inside the requirements.txt file itself? Don't understand why we don't pin down the version for these 2 packages.